packages/sacado/src/new_design/Sacado_Fad_Exp_GeneralFad.hpp

```diff
@@ -31,6 +31,7 @@
 #define SACADO_FAD_EXP_GENERALFAD_HPP
 
 #include "Sacado_Fad_Exp_GeneralFadTraits.hpp"
+#include "Sacado_Fad_GeneralFadTestingHelpers.hpp"
```

Comment:
This is definitely not the right place to include this.  Actually, I'm not sure any Sacado header should include this by default.  Instead I think any testing code that uses it should include it.